docs/index.html

```diff
@@ -85,7 +98,7 @@
     <style>
       :root {
         /* Reduce the font size */
-        --base-font-size: 14px;
```

Comment:
I also removed the custom font size to go back to the default Docsify font size of 16px as it is more standard